### DISCUSSION TITLE
Covid19 segmentation

### DISCUSSION THREAD

**Discussion by kirmanserdar** [Project-MONAI/MONAI#2110] [General]
Hi 👋👋

Is there any MONAI tutorial/work about covid19 ct images segmentation with 3D Unet ?(especially,  [this dataset)](https://zenodo.org/record/3757476#.YIppY7X7SUn)

tutorial will be of great help

**Comment by masadcv** [comment]
Hi @kirmans 
Have you checked this one: https://github.com/Project-MONAI/tutorials/tree/master/3d_segmentation/challenge_baseline
It uses data from covid grand challenge: https://covid-segmentation.grand-challenge.org/
You can modify the data loaders to load your own dataset to train it.
